ct/cronicle.sh

```diff
@@ -53,13 +53,15 @@ function default_settings() {
 }
 
 function update_script() {
+header_info
+check_container_storage
+check_container_resources
 UPD=$(whiptail --backtitle "Proxmox VE Helper Scripts" --title "SUPPORT" --radiolist --cancel-button Exit-Script "Spacebar = Select" 11 58 2 \
-  "1" "Update ${APP}" ON \
+  "1" "Upgrade ${APP}" ON \
```

Comment:
```suggestion
  "1" "Update ${APP}" ON \
```
Why change, we call it update everywhere else